src/main/java/org/folio/linked/data/util/BibframeUtils.java

```diff
@@ -121,17 +118,15 @@ public static Resource ensureLatestReplaced(Resource resource) {
       .orElse(resource);
   }
 
-  public static Resource fetchResource(Identifiable identifiable, ResourceRepository resourceRepository) {
+  public static Optional<Resource> fetchResource(Identifiable identifiable, ResourceRepository resourceRepository) {
```

Comment:
As mentioned in previous comment, we could move function to [ResourceMarcAuthorityServiceImpl.java](https://github.com/folio-org/mod-linked-data/pull/39/files#diff-85288c11e1be3c1085c3b6639e757e8fa526aac502ca9446df51fb763f7640f5)

Reply:
done.